sweep/sweeper.go

```diff
@@ -978,49 +959,6 @@ func (s *UtxoSweeper) markInputsPublishFailed(set InputSet) {
 	}
 }
 
-// monitorSpend registers a spend notification with the chain notifier. It
-// returns a cancel function that can be used to cancel the registration.
-func (s *UtxoSweeper) monitorSpend(outpoint wire.OutPoint,
-	script []byte, heightHint uint32) (func(), error) {
-
-	log.Tracef("Wait for spend of %v at heightHint=%v",
-		outpoint, heightHint)
-
-	spendEvent, err := s.cfg.Notifier.RegisterSpendNtfn(
-		&outpoint, script, heightHint,
-	)
-	if err != nil {
-		return nil, fmt.Errorf("register spend ntfn: %w", err)
-	}
-
-	s.wg.Add(1)
-	go func() {
```

Comment:
So perhaps we'll retain some form of this after all?

Reply:
Agree - so I ended up not touching this area in this PR to limit the scope. For now we would keep this spend notification in the sweeper and read them in a goroutine to make sure we won't miss it.